sktime/forecasting/model_selection/tests/test_tune.py

```diff
@@ -329,14 +333,11 @@ def test_skoptcv_multiple_forecaster():
     assert len(sscv.cv_results_) == 5
 
 
-@pytest.mark.xfail
 @pytest.mark.skipif(
     not run_test_for_class(ForecastingOptunaSearchCV),
     reason="run test only if softdeps are present and incrementally (if requested)",
 )
-@pytest.mark.parametrize(
-    "forecaster, param_grid", [(NAIVE, NAIVE_GRID), (PIPE, PIPE_GRID)]
```

Comment:
You could also add in the Pipe and Pipe_Grid using the correct distributions for Optuna

Reply:
Good idea, did that as well